### PR TITLE
Increase EFI partition size to 1M

### LINKED ISSUE #82: Cannot write uefi-ntfs.img to uefi partition
## Describe the bug ##

Cannot write uefi-ntfs.img to uefi partition, not enough space. The USB stick has 14,9G, more than enough, but the UEFI partition is created with only 512k. As the image `uefi-ntfs.img` has ~1M, it gives this error.

## To Reproduce ##

1. `sudo ./woeusb-5.1.2.bash --device ~/Downloads/Win10_21H1_BrazilianPortuguese_x64.iso /dev/sdb --tgt-fs NTFS`
2. See error

## Expected behavior ##
Bootable device to be created

## Teminal output ##
```
2021-10-30 09:57:57 (2,84 MB/s) - “/tmp/WoeUSB.tempdir.NGByCR/uefi-ntfs.img” saved [1048576/1048576]

dd: writing in '/dev/sdb2': Not enough space on device
1025+0 registros de entrada
1024+0 registros de saída
524288 bytes (524 kB, 512 KiB) copiados, 0,122135 s, 4,3 MB/s
ERROR: The command "dd if="${download_directory}/uefi-ntfs.img" of="${uefi_ntfs_partition}"" failed with exit status "1", program is prematurely aborted
```

## Environment ##

- OS: Arch Linux
- Bash: 5.1.8(1)-release
- WoeUSB: 5.1.2

### PR DESCRIPTION
**Pull request by philenotfound**
Image uefi-ntfs.img does not fit anymore otherwise

Fixes #82, #81

Signed-off-by: Phil Eichinger <phil@zankapfel.net>

### CODE REVIEW

**Comment by WaxyMocha** [comment]
@brlin-tw This needs yours attention.

File: sbin/woeusb [hunk 1]
```diff
@@ -1047,7 +1047,7 @@ create_uefi_ntfs_support_partition(){
         primary \
         fat16 \
         --  \
-        -1024s \
+        -2048s \
```
Comment (brlin-tw) [inline sbin/woeusb:1050]: I am looking to add support to calculate the numbers in run-time, but this would work for now.

**Comment by brlin-tw** [comment]
Apologies for the ignorance, although I have already been notified about this issue I have no capacity to check it out...until now.

Also thanks to @WaxyMocha for the heads up, it is appreciated.

**Comment by brlin-tw** [comment]
The patch is been merged and should be available in [the WoeUSB 5.1.3 release](https://github.com/WoeUSB/WoeUSB/releases/tag/v5.1.3).

**Pull request merged**